guarddog/cli.py

```diff
@@ -165,14 +165,18 @@ def _scan(identifier, version, rules, exclude_rules, output_format, exit_non_zer
     if scanner is None:
         sys.stderr.write(f"Command scan is not supported for ecosystem {ecosystem}")
         exit(1)
-    results = {}
+    results = []
     if is_local_target(identifier):
         log.debug(f"Considering that '{identifier}' is a local target, scanning filesystem")
```

Comment:
can we add a log.debug statement to show when the local input is considered a file or a directory?

Reply:
Done